pkg/webhook/server.go

```diff
@@ -53,10 +48,15 @@ type Server struct {
 	Port int
 
 	// CertDir is the directory that contains the server key and certificate. The
-	// server key and certificate must be named tls.key and tls.crt,
-	// respectively.
+	// server key and certificate.
 	CertDir string
 
+	// CertName is the server certificate name. defaults to tls.crt
```

Comment:
NIT: Choose the one you like most:

```suggestion
	// CertName is the server certificate name. Defaults to tls.crt.
```
```suggestion
	// CertName is the server certificate name, defaults to tls.crt.
```